docker/docker-compose.dev.rosetta-cli.yml

```diff
@@ -4,6 +4,6 @@ services:
     build:
       context: ../rosetta-cli-config
       dockerfile: docker/Dockerfile
-    command: ${CMD}
+    command: /bin/rosetta-cli --configuration-file /app/rosetta-config-docker.json check:construction
     volumes:
-      - ${OUTPUT}
+      - ./rosetta-output-construction:/app/rosetta-output
```

Comment:
The only intended purpose of this dockerfile is for integration tests, which this change appears will break: https://github.com/hirosystems/stacks-blockchain-api/blob/9dc72488bff40de008d4245172259185840fc670/src/tests-rosetta-cli-construction/validate-construction.ts#L45

It sounds like from issue https://github.com/hirosystems/stacks-blockchain-api/issues/1555 that you're trying to get `npm run dev:integrated` to work? As Rafael mentioned, this is a very old command that should be removed. 

Can you give more context into what you are trying to achieve? Then we can point you towards the intended workflow, or more appropriate areas to modify.

Reply:
Also note that when you change target git branches for a PR, you need to rebase your commit on top the new branch. Otherwise you end up with a PR file diff that has changes other than your own.

Reply:
@zone117x As you pointed out the issue #1555, the way I thought to remove the command was to replace it with the updated one. If this breaks the integration tests, this PR stands NULL, and I will close it with your confirmation. 

I tried to rebase the changes on top of my changes but couldn't achieve that. I will look into it for future reference. Thanks for your help.
Also, could you please suggest any open issue that I could work on? I wanna contribute to the repo.